Tests/SonarScanner.MSBuild.PreProcessor.Test/ArgumentProcessorTests.cs

```diff
@@ -584,13 +585,13 @@ public void PreArgProc_SkipJreProvisioning_SetInvalid(string skipJreProvisioning
         #region Checks
 
         private static TestLogger CheckProcessingFails(params string[] commandLineArgs) =>
-            CheckProcessingFails(Substitute.For<IFileWrapper>(), commandLineArgs);
+            CheckProcessingFails(Substitute.For<IFileWrapper>(), Substitute.For<IDirectoryWrapper>(), commandLineArgs);
 
-        private static TestLogger CheckProcessingFails(IFileWrapper fileWrapper, params string[] commandLineArgs)
+        private static TestLogger CheckProcessingFails(IFileWrapper fileWrapper, IDirectoryWrapper directoryWrapper, params string[] commandLineArgs)
         {
             var logger = new TestLogger();
 
-            var result = TryProcessArgsIsolatedFromEnvironment(commandLineArgs, fileWrapper, logger);
+            var result = TryProcessArgsIsolatedFromEnvironment(commandLineArgs, fileWrapper, directoryWrapper, logger);
```

Comment:
All the checks seem to use an unconfigured `Substitute.For<IDirectoryWrapper>()`. Can't we use the value directly here?

Reply:
I added tests in fe337c68e3a7fed9fd7346e27d6efbc5962d16c8 that use the wrapper. These UTs were missing anyway.